dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -487,10 +487,17 @@ const backgroundHeadlineByline = (format: ArticleFormat): string => {
 };
 
 const backgroundBullet = (format: ArticleFormat): string => {
-	if (format.theme === ArticleSpecial.Labs) return BLACK;
-	if (format.theme === ArticleSpecial.SpecialReport)
-		return specialReport[300];
-	return pillarPalette[format.theme].main;
+	switch (format.design) {
+		case ArticleDesign.DeadBlog:
+			return neutral[60];
+		case ArticleDesign.LiveBlog:
```

Comment:
I chose this abstraction because bullets for liveblogs will vary depending on theme